modules/apigw/outputs.tf

```diff
@@ -6,6 +6,14 @@ output "url" {
   value = aws_api_gateway_deployment.deployment.invoke_url
 }
 
+output "url_hostname" {
+  value = element(regex("^https://(.*com)/.*", aws_api_gateway_deployment.deployment.invoke_url), 0)
```

Comment:
Will it always be `https`?

Reply:
Yes I think, api gateway only allows `https` connections.  But good point though, maybe it's better to change the regex to accept both http and https just to be safe

Reply:
Based on the comment (https://github.com/DazKins/tf-lambda-proxy-apigw/pull/1#discussion_r451604699) from David, this won't matter for now actually. When you use this regex in another module to extract the host, I'm fine with `https` if that's the only valid option.

Reply:
OK done. https://github.com/efcloud/el-b2c-environments/pull/364/files#diff-0a777460f85b59221c0ea1c9824c9d9dR20

Reply:
I can't resolve this comment but I'm happy with it